compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp

```diff
@@ -577,13 +577,29 @@ bool QuantizeWithMinMaxPass::run(loco::Graph *g)
     return _ctx->granularity;
   };
 
+  // to save min and max values
+  std::unordered_map<luci::CircleNode *, std::vector<float>> node_to_min_values;
+  std::unordered_map<luci::CircleNode *, std::vector<float>> node_to_max_values;
+
   // Quantize activation
   // Why all_nodes?
   // Models can have inactive (unused) inputs.
   // We do not reject such models, but quantize them too
   for (auto node : loco::all_nodes(g))
   {
     auto circle_node = loco::must_cast<luci::CircleNode *>(node);
+
+    // Save min max values
+    if (_ctx->save_min_max)
+    {
+      const auto quant_param = circle_node->quantparam();
+      if (quant_param != nullptr)
+      {
+        node_to_min_values[circle_node] = quant_param->min;
+        node_to_max_values[circle_node] = quant_param->max;
+      }
+    }
```

Comment:
Q) Is there a reason to save min/max values here and restore later? Isn't it possible to maintain min/max values and remove them only when `_ctx->save_min_max = true`?

Reply:
As I understand there are special passes (https://github.com/Samsung/ONE/blob/11478d7ccffbc3bd55eedfe1a4d1498ba992c808/compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp#L629-L642) for some operations, that can remove forget about min,max values. 
I thought there might be more such places where values are implicitly forgotten, so I decided to keep it that way for sure, but if it would be better to change the methods themselves that create new quantum parameters (for example `make_predefined_qparam` and `quant_const`), then I will do so.
 Probably the second option is to change these methods so that they always save and then at the end, depending on the value of the flag, either delete or save these values.

Reply:
The previous way saves a map (node pointer -> min/max values) at the beginning. But the saved pointers can be invalidated, because nodes can be removed and newly created during quantization. 

I see that you already made a change. Thanks!